src/ploomber/tasks/notebook.py

```diff
@@ -532,6 +586,10 @@ def __init__(self,
         self.nb_product_key = nb_product_key
         self.local_execution = local_execution
         self.check_if_kernel_installed = check_if_kernel_installed
+        self.multiple_nb_params = {
+            'valid_keys': ['nb_ipynb', 'nb_html', 'nb_pdf'],
+            'ipynb_key': 'nb_ipynb'
+        }
```

Comment:
this should be defined by the user, for example, a user might call this: `[notebook, report_html, report_pdf]`, then the validation should be done on those user-submitted values. Looks like this code is looking for a `nb_ipynb` key.

For example, I may wanna do this:

```yaml
  - source: fit.py
    nb_product_key: [notebook, nb_html, nb_pdf]
    product:
      notebook: output/report.ipynb
      nb_html: output/report.html
      nb_pdf: output/report.pdf
      model: output/model.pickle
```

Or may not even want the ipynb file:

```yaml
  - source: fit.py
    nb_product_key: [report_web, report_document]
    product:
      report_web: output/report.html
      report_document: output/report.pdf
      model: output/model.pickle
```

Reply:
@edublancas 
Made the requested changes. Do let me know if anything looks incorrect